### PR TITLE
Add an endpoint for displaying the public XML

### PR DESCRIPTION
**Pull request by mjgiarlo**
## Why was this change made?

Connects to sul-dlss/infrastructure-integration-test#241

## How was this change tested?

CI

## Which documentation and/or configurations were updated?

none

### CODE REVIEW

**Review by ndushay — APPROVED** [review]

File: spec/requests/metadata_spec.rb [hunk 1]
```diff
@@ -48,4 +48,75 @@
...
+          <identityMetadata/>
+          <rightsMetadata>
+            <access type="discover">
+              <machine>
+                <none/>
+              </machine>
+            </access>
+            <access type="read">
+              <machine>
+                <none/>
```
Comment (jmartin-sul) [inline spec/requests/metadata_spec.rb:91]: it seems odd to me to have public XML for a dark object (read and discover both being none is dark, right?), but i don't know enough to know whether that's actually something we shouldn't produce, so probably fine, esp in a unit test? 🤷

**Review by jmartin-sul — APPROVED** [review]
noticed one thing that surprised me, but don't know enough to know whether it's actually wrong.  LGTM, approving, feel free to merge if my question is of no concern.

File: spec/requests/metadata_spec.rb (hunk 1 above)
Comment (mjgiarlo) [reply spec/requests/metadata_spec.rb:91]: @jmartin-sul Yeah, I don't think the public xml service validates that itself. We could "fix" this with more mocking. Worth the effort or nah?

Comment (jmartin-sul) [reply spec/requests/metadata_spec.rb:91]: nah

**Pull request merged**